bsp/boards/iot-lab_A8-M3/board.c

```diff
@@ -96,8 +101,48 @@ void board_init(void){
     debugpins_init();
     //enable nvic for the radio
     NVIC_radio();
+    
+    board_init_slot_vars();
+}
+
+
+//==== bootstrapping slot info lookup table
+void board_init_slot_vars(void){
+    
+    // 20ms slot
+    slot_board_vars [SLOT_20ms_24GHZ].PORT_SLOTDURATION                   =  20   ; // ms  
+    slot_board_vars [SLOT_20ms_24GHZ].PORT_TsSlotDuration                 =  655  ; //    20ms
+    slot_board_vars [SLOT_20ms_24GHZ].PORT_maxTxDataPrepare               =  110  ; //  3355us (not measured)
+    slot_board_vars [SLOT_20ms_24GHZ].PORT_maxRxAckPrepare                =  20   ; //   610us (not measured)
+    slot_board_vars [SLOT_20ms_24GHZ].PORT_maxRxDataPrepare               =  33   ; //  1000us (not measured)
+    slot_board_vars [SLOT_20ms_24GHZ].PORT_maxTxAckPrepare                =  50   ; //  1525us (not measured)
+    slot_board_vars [SLOT_20ms_24GHZ].PORT_delayTx                        =  18   ; //   549us (not measured)
+    slot_board_vars [SLOT_20ms_24GHZ].PORT_delayRx                        =  0    ; //     0us (can not measure)
 }
 
+// To get the current slotDuration at any time
+// used during initialization by sixtop to fire the first sixtop EB
+uint16_t board_getSlotDuration (time_type_t time_type)
+{
+  if (time_type == TIME_MS)
+    return slot_board_vars [selected_slot_type].PORT_SLOTDURATION;

```

Comment:
you cannot keep maintaining 2 variables with 2 units. Kill the "ms" variables, keep only the one in ticks